e2e/tests/functional/plugins/imagery/exampleImagery.e2e.spec.js

```diff
@@ -573,6 +573,42 @@ test.describe('Example Imagery in Tabs view', () => {
     test.fixme('If the imagery view is not in pause mode, it should be updated when new images come in');
 });
 
+test.describe('Example Imagery in Time Strip', () => {
+    test('ensure that clicking a thumbnail loads the image in large view', async ({ page, browserName }) => {
+        test.info().annotations.push({
+            type: 'issue',
+            description: 'https://github.com/nasa/openmct/issues/5632'
+        });
+        await page.goto('./', { waitUntil: 'networkidle' });
+        const timeStripObject = await createDomainObjectWithDefaults(page, {
+            type: 'Time Strip',
+            name: 'Time Strip'.concat(' ', uuid())
+        });
+
+        await createDomainObjectWithDefaults(page, {
+            type: 'Example Imagery',
+            name: 'Example Imagery'.concat(' ', uuid()),
+            parent: timeStripObject.uuid
+        });
+
+        await page.goto('./#/browse/mine?hideTree=false');
+        // expand root folder
+        await page.locator('text=Open MCT My Items >> span').nth(3).click();
+        await page.locator(`.c-tree__item a:has-text("${timeStripObject.name}")`).click();
```

Comment:
I think in this case we can actually avoid the tree entirely by doing this:

```js
await page.goto(timeStripObject.url);
```

If for whatever reason we do need the tree expanded (for example, if you need to drag and drop an object from the tree), I like to use the 'target' button (show selected item in tree) like this:

```js
// First navigate to the object you want to drag and drop
await page.goto(timeStripObject.url);
// Expand the tree to reveal the selected object
await page.click('button[title="Show selected item in tree"]');
```

Reply:
I like your approach much more. Will update the test to match.